src/Controller/Utils/StoreTrait.php

```diff
@@ -41,21 +41,33 @@ trait StoreTrait
 {
     abstract protected function getStoreList();
```

Comment:
This method is not used anymore now, it can be removed. 
Also remove the concrete implementations in `DashboardController` & `AdminController`